### ISSUE TITLE
Add type to MediaObject

### ISSUE THREAD

**Issue by sekaiking** [curaOS/source#189]
After this https://github.com/curaOS/contracts/issues/3 gets implemented we should add `type="image"` to `MediaObject` components in the frontend, because now it assume all medias are HTML

And because of that, they get displayed wrong
![image](https://user-images.githubusercontent.com/54310457/146633071-796d7f00-1921-4153-bc34-14b3b26c2e98.png)

**Comment by devey** [comment]
It's working fine at the moment? @achrafismyname

**Comment by achrafismyname** [comment]
Yes and no, some NFTs got minted using this PR #194 and are causing this problem. 
But it's not a big problem, this mainly need to be implemented after #194 gets merged.
